whois/whois.py

```diff
@@ -41,8 +41,23 @@ def __init__(self, bot):
         results = [
             (ctx.guild.get_member(int(userid)), name) for userid, name in results
         ]
+
+        CHAR_LIMIT = 2000
```

Comment:
change this to not uppercase - uppercase vars are reserved for globals not locals